<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="51566" started-at="2016-03-02T07:04:59Z" finished-at="2016-03-02T07:05:50Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="51566" started-at="2016-03-02T07:04:59Z" finished-at="2016-03-02T07:05:50Z">
      <class name="ALLie_AutoTests.Restore">
        <test-method status="PASS" signature="setUp()[pri:0, instance:ALLie_AutoTests.Restore@6adca536]" name="setUp" is-config="true" duration-ms="14607" started-at="2016-03-02T12:04:59Z" finished-at="2016-03-02T12:05:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="Default()[pri:0, instance:ALLie_AutoTests.Restore@6adca536]" name="Default" duration-ms="35749" started-at="2016-03-02T12:05:13Z" finished-at="2016-03-02T12:05:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Default -->
        <test-method status="PASS" signature="catchExceptions(org.testng.ITestResult)[pri:0, instance:ALLie_AutoTests.Restore@6adca536]" name="catchExceptions" is-config="true" duration-ms="1162" started-at="2016-03-02T12:05:49Z" finished-at="2016-03-02T12:05:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Default status=SUCCESS method=Restore.Default()[pri:0, instance:ALLie_AutoTests.Restore@6adca536] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- catchExceptions -->
      </class> <!-- ALLie_AutoTests.Restore -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
